geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/ingest/AbstractIngest.scala

```diff
@@ -94,6 +94,18 @@ abstract class AbstractIngest(val dsParams: Map[String, String],
     ds.dispose()
   }
 
+  def getTypeAndFeatures(file: File, failed: AtomicLong): (SimpleFeatureType, Iterator[SimpleFeature], CountingInputStream, Closeable) = {
+    val converter = createLocalConverter(file, failed)
+    // count the raw bytes read from the file, as that's what we based our total on
+    val countingStream = new CountingInputStream(new FileInputStream(file))
+    val is = PathUtils.handleCompression(countingStream, file.getPath)
+    val (sft, features) = converter.convert(is)
+    val closeable = new Closeable {
+      override def close(): Unit = { converter.close(); is.close() }
```

Comment:
you need to put this in closeQuietly like before to ensure the is.close() happens...